examples/forward/plot_forward_sensitivity_maps.py

```diff
@@ -28,7 +28,8 @@
 subjects_dir = data_path + '/subjects'
 
 # Read the forward solutions with surface orientation
-fwd = mne.read_forward_solution(fwd_fname, surf_ori=True)
+fwd = mne.read_forward_solution(fwd_fname)
+fwd = mne.convert_forward_solution(fwd, surf_ori=True)
```

Comment:
For reading a free, surface-oriented forward (`surf_ori=True`, `force_fixed=False`) `use_cps=None` is equivalent to `use_cps=True`, that's why I didn't give `use_cps` explicitely here. The warning should only pop up if `force_fixed=True`, as only there the default will be changed.